e2e-test/cbas_azure_e2e_test.py

```diff
@@ -0,0 +1,219 @@
+from workspace_helper import create_workspace
+from app_helper import create_app, poll_for_app_url
+import requests
+import os
+import json
+import random
+import string
+import uuid
+import time
+
+
+# Setup configuration
+# These values should be injected into the environment before setup
+azure_token = os.environ.get("AZURE_TOKEN")
+bee_name = os.environ.get("BEE_NAME")
+billing_project_name = os.environ.get("BILLING_PROJECT_NAME")
+
+rawls_url = f"https://rawls.{bee_name}.bee.envs-terra.bio"
+leo_url = f"https://leonardo.{bee_name}.bee.envs-terra.bio"
+
+
+# Upload data to WDS
+def upload_wds_data(wds_url, workspace_id, tsv_file_name, record_name):
```

Comment:
Is this different from the `upload_wds_data` in `helper.py`?

Reply:
The method here is using API as compared to WDS client to upload the data. I was running into problems with WDS client and since other methods were also using direct APIs I opted to use direct APIs for WDS too.

Reply:
Ok, can you add some comments (or even rename the method) just so that's clear to anyone else who notices that there appears to be duplicate methods?  Also, what problems were you running into with the client?  Maybe that's something I should look into.

Reply:
Yes I can rename the method and add comment to specify the use of APIs. I don't remember the exact error but it was mostly around not being able to install the WDS client. I actually didn't investigate too much and switched to APIs as direct APIs were being used everywhere else.